yarn.lock

```diff
@@ -5504,6 +5504,27 @@
     "@walletconnect/window-metadata" "1.0.0"
     detect-browser "5.2.0"
 
+"@walletconnect/browser-utils@^1.7.8":
```

Comment:
Any idea what caused these updates? No package changes to trigger it

Reply:
nup, my assumption is that it was not updated.... i didn't do anything else than running `yarn`